egs2/libritts/speechlm1/conf/slurm.conf

```diff
@@ -0,0 +1,16 @@
+# Default configuration
+command sbatch --export=PATH
+option name=* --job-name $0
+default time=48:00:00
+option time=* --time $0
+default mem=2000
+option mem=* --mem-per-cpu $0
+option mem=0
+option num_threads=* --cpus-per-task $0
+option num_threads=1 --cpus-per-task 1
+option num_nodes=* --nodes $0
+default gpu=0
+option gpu=0 -p swl_general --exclude=swl-2-[5,9]
+option gpu=* -p swl_general --gres=gpu:$0 -c 5 --exclude=swl-2-[5,9] # Recommend allocating more CPU than, or equal to the number of GPU
```

Comment:
let's change back to default for the pr.

Reply:
sure.